tmt/steps/provision/mrack/mrack-provisioning-config.yaml

```
@@ -109,7 +111,9 @@ beaker:
     timeout: 230 # in minutes
 
     tasks:
-        default: [ {"name": "/distribution/dummy", "role": "STANDALONE"} ]
+        default:
+            - name: /distribution/dummy
+              role: STANDALONE
```

```suggestion
          - name: /distribution/dummy
            role: STANDALONE
```

Shouldn't this be two spaces less? We consistently use four spaces for list content, not for the `-` character. This is also how `.pre-commit-config.yaml` is formatted and `yamllint` seems to be happy there. But not here:

```
tmt/steps/provision/mrack/mrack-provisioning-config.yaml
  115:11    error    wrong indentation: expected 12 but found 10  (indentation)
```

Is this some special case?

---

Right, this was caused by the yamllint's defaults, "consistent" indentation - whatever indentation one uses for the first time in the file, that would be enforced in the rest of the file. And `mrack-provisioning-config.yaml` used 4 spaces.

Enforcing 2 space indentation in 2b2b00ae - more changes in `mrack-provisioning-config.yaml`, but it's aligned with the rest of the files covered by yamllint.

---

There is also another option, drop the yamllint effort and try a genuine code formatter, like https://prettier.io/ - but I did not try it enough to find out whether it has limitations of its own.